.gitignore

```diff
@@ -19,3 +19,7 @@
 fixtures/
 .env
 .secrets
+
+# API Testing
+.access_token_cached
+api.py
```

Comment:
Some Python scripts I was using to test and debug what I was getting from Auth0 and what the Golang code code expect and parse. I'm happy to provide this script as a private gist on request.